packages/nimble-components/build/generate-workers/source/matrix-renderer.ts

```diff
@@ -8,16 +9,158 @@ import { expose } from 'comlink';
  * This setup is used in the wafer-map component to perform heavy computational duties
  */
 export class MatrixRenderer {
-    public dieMatrix: Uint8Array = Uint8Array.from([]);
+    public colIndexes: Int32Array = Int32Array.from([]);
+    public rowIndexes: Int32Array = Int32Array.from([]);
+    public values = new Float64Array([]);
+    public scaledColIndex = new Float64Array([]);
```

Comment:
Why are we using two different ways to initialize an empty array:
~~~
public colIndex: Int32Array = Int32Array.from([]);
and
public scaledColIndex = new Float64Array([]);
~~~

Additionally, this should be called `scaledColumnIndexes` (and below would be `scaledRowIndexes`).

Reply:
The difference between Int32Array.from([]) and new Int32Array([]) can be observed in a JS/TS environment that is not strictly configured, Int32Array.from([]) returns for sure an Int32Array while new Int32Array([]) because of the empty parameter it might be seen as a regular JS array and be inferred to an regular JS array (not sure why, didn't get deep into it). The different ways of declaration in code though doesn't have an explanation, I was experimenting with different kind of declaration when things weren't working and were unintentionally left. I will use .from([]) for all. Regarding the naming, I will rename the property.